doc/config.rst

```diff
@@ -1663,6 +1663,16 @@ These options influence LaTeX output. See further :doc:`latex`.
      ``'maketitle'``
         "maketitle" call, default ``'\\maketitle'``.  Override if you want to
         generate a differently-styled title page.
+     ``'fvset'``
```

Comment:
@tk0miya do you mean that ``\fvset{fontsize=\small}`` should be part of ``sphinx.sty`` and advanced user can still do in ``preamble`` key own setting for ``\fvset`` ? Perhaps then move mention of ``\fvset`` to the new ``latex.rst``. You suggest suppression of ``'fvset'`` key ?